books/kestrel/c/syntax/input-files-doc.lisp

```diff
@@ -131,35 +131,41 @@
       "If @(':preprocess') is @('nil'),
        the @(':preprocess-args') input must be absent.")
      (xdoc::p
-      "If @(':preprocess') is not @('nil'),
-       and the @(':preprocess-args') input is absent,
-       the arguments @('-E') and @('-P') are passed to the preprocessor,
-       in that order.")
-     (xdoc::p
-      "If @(':preprocess') is not @('nil'),
-       and @(':preprocess-args') input is present,
-       the behavior depends on whether
-       it evaluates to a string list or to an omap.
-       For each file, the argument @('-E') is passed to the preprocessor.
+      "If @(':preprocess') is not @('nil'), the input files are preprocessed.
+       For each file, we provide the @('-E') flag to the preprocessor,
+       as well as some instantiation of the @('-std=') flag.
+       This latter flag is based on the combination of the
+       @(':std') and @(':gcc') inputs,
+       as indicated by the following table.")
+     (xdoc::table_
+      (xdoc::tr (xdoc::th)
+                (xdoc::th "@(':gcc nil')")
+                (xdoc::th "@(':gcc t')"))
+      (xdoc::tr (xdoc::th "@(':std 17')")
+                (xdoc::td "@('-std=c17')")
+                (xdoc::td "@('-std=gnu17')"))
+      (xdoc::tr (xdoc::th "@(':std 23')")
+                (xdoc::td "@('-std=c23')")
+                (xdoc::td "@('-std=gnu23')")))
+     (xdoc::p
+      "When present, the @(':preprocess-args') specifies additional arguments
```

Comment:
```suggestion
      "The @(':preprocess-args') specifies additional arguments
```

Reply:
Now that there is a default, its absence is equivalent to its presence with the default.